### PR TITLE
(RE-3217) Add an 'RE' ticket to the release tickets

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by melissa** [comment]
I'm having issues testing this at the moment. There are issues connecting to the test server. I'll wait until tomorrow to try again.

File: tasks/tickets.rake [hunk 1]
```diff
@@ -393,6 +393,7 @@ DOC
 
   # Create subtasks for each step of the release process
   subticket_idx = 1
+  release_tickets = []
   subtickets.each { |subticket|
```
Comment (haus) [inline tasks/tickets.rake]: { => do

File: tasks/tickets.rake [hunk 2]
```diff
@@ -404,8 +405,23 @@ DOC
 
     puts "\tSubticket #{subticket_idx.to_s.rjust(2)}: #{key} (#{subticket[:assignee]}) - #{subticket[:summary]}"
 
+    release_tickets << key if subticket[:assignee] == vars[:builder]
+
     subticket_idx += 1
   }
```
Comment (haus) [inline tasks/tickets.rake]: } => end

**Comment by melissa** [comment]
https://jira1-test.ops.puppetlabs.net/browse/RE-5490 is the example ticket for this work. It's supposed to only block against tickets that were assigned to the builder, but since I assigned myself to everything, it blocked against a lot more :)

**Comment by haus** [comment]
:+1:

**Pull request merged**